### PR TITLE
PR to merge Feature into develop

### PR DESCRIPTION
**Pull request by UmairAhmad125**
In this project(private events), 
- We build associations like one-to-one, one-to-many, and many-to-many. 
- Added validations 
- More details are mentioned in the readme file

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
## Status Changes required
Really great work with this project @UmairAhmad125  @talhawaqar  your ui looks lovely 👍 
 
Below are some missing features to implement.

- [ ] There currently is no way to invite a user to an event.
User as an Event attendee ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/attendee_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

**Comment by UmairAhmad125** [comment]
> ## Status Changes required
> Really great work with this project @UmairAhmad125 @talhawaqar your ui looks lovely 
> 
> Below are some missing features to implement.
> 
> * [ ]  There currently is no way to invite a user to an event.
>   User as an Event attendee ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/attendee_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

All the required associations are there
Invite a user is an [extra credit](https://ibb.co/6RnM27h) task

File: README.md [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
 
-* ...
+:bust_in_silhouette: **Umair Ahmad**
+
+- Github: [@umairahmad125](https://github.com/UmairAhmad125)
+- Linkedin: https://www.linkedin.com/in/umair-ahmad-b5a89015a/
+- Twitter: [@umairahmadDP](https://twitter.com/umairahmadDP)
+- email: (umairahmad125@gmail.com)
+  :bust_in_silhouette: **Muhammad Talha Waqar**
+- Github: [@talhawaqar](https://github.com/talhawaqar)
```
Comment (mosesogwo) [inline README.md:39]: Check the appearance of this section of your README.

**Review by mosesogwo — APPROVED** [review]
**Hello:wave: Well done on this:clap: Your project is approved**

 Please check your specs. You have one failing test.

### Status: Approved :heavy_check_mark:

**Pull request merged**